File: backend/WebRtcApi/Models/MentorPackage.cs
using System;
using System.Collections.Generic;

namespace WebRtcApi.Models;

public partial class MentorPackage
{
    public int PackageId { get; set; }

    public string Name { get; set; } = null!;

    public string? Description { get; set; }

    public decimal Price { get; set; }

    public int DurationMonths { get; set; }

    public int? CreatedBy { get; set; }

    public DateTime? CreatedAt { get; set; }

    public virtual User? CreatedByNavigation { get; set; }

    public virtual ICollection<Transaction> Transactions { get; set; } = new List<Transaction>();
}

File: frontend/admin/src/types/mentorPackage.ts
export interface Package {
  packageId: number;
  name: string;
  description?: string | null;
  price: number;
  durationMonths: number;
  createdAt: string;
  createdBy?: {
    id: number;
    name: string;
    avatar?: string | null;
  } | null;
}

export interface CreatePackageInput {
  name: string;
  description?: string;
  price: number;
  durationMonths: number;
}

export interface UpdatePackageInput {
  name?: string;
  description?: string;
  price?: number;
  durationMonths?: number;
}


